.gitignore

```
@@ -6,3 +6,7 @@ work/
 
 # ignore scpca-references
 scpca-references/
+
+
+# ignore template htmls
+templates/qc_report/*html
```

I don't know if we might expect these files to be littered about other places, so I made this both more specific and more general. Also an extra space. Not sure what the _best_ option is here, but this was more what I was thinking for moving to root `.gitignore`.
```suggestion
qc_report.html
*_qc.html

```